specification/vmwarecloudsimple/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,13 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+```yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "vmwarecloudsimple"
```

Comment:
This appears to be a management-plane package, so please prefix the package name with `@azure/arm-`.
```suggestion
  package-name: "@azure/arm-vmwarecloudsimple"
```